<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
 <class>MeshcontrolForm</class>
 <widget class="QDialog" name="MeshcontrolForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../ElmerGUI.qrc" >
    <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="engineGroup" >
     <property name="title" >
      <string>Preferred generator</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="tetlibRadioButton" >
        <property name="text" >
         <string>tetlib (.stl .smesh .poly .off .ply .mesh)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="nglibRadioButton" >
        <property name="text" >
         <string>nglib (.in2d .stl .brep .step .stp .iges .igs)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="elmerGridRadioButton" >
        <property name="text" >
         <string>elmergrid (.grd .FDNET .msh .mphtxt .unv)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tetlibGroup" >
     <property name="title" >
      <string>tetlib</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="tetlibStringLabel" >
        <property name="minimumSize" >
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>String:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="tetlibStringEdit" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="nglibGroup" >
     <property name="title" >
      <string>nglib</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="nglibMaxHLabel" >
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Max H:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="nglibMaxHEdit" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="nglibFinenessLabel" >
        <property name="minimumSize" >
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Fineness:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="nglibFinenessEdit" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="nglibBgmeshLabel" >
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Bgmesh:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="nglibBgmeshEdit" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="elmerGridGroup" >
     <property name="title" >
      <string>elmergrid</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="elmerGridStringLabel" >
        <property name="minimumSize" >
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>String:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="elmerGridStringEdit" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="elementCodesGroup" >
     <property name="title" >
      <string>Element codes (for solver):</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="elementCodesLabel" >
        <property name="minimumSize" >
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Codes:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="elementCodesStringEdit" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="defaultsButton" >
       <property name="text" >
        <string>Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Apply</string>
       </property>
       <property name="icon" >
        <iconset theme="dialog-ok-apply" />
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ElmerGUI.qrc" />
 </resources>
 <connections/>
</ui>
